mda_openbabel_converter/OpenBabelParser.py

```diff
@@ -3,27 +3,286 @@
+                    " which are NOT supported by MDAnalysis.")
+            charges.append(atom.GetPartialCharge()) #int (or use atom.GetFormalCharge()?)
+
+            # convert atomic number to element
+            elements.append(OBElementTable().GetSymbol(atom.GetAtomicNum())) #char
+
+            if atom.HasResidue():
+                resid = atom.GetResidue() # null if no residue
+                resnums.append(resid.GetNum()) # TO DO: check if start at 0 or 1
+                resnames.append(resid.GetName())
+                chainids.append(resid.GetChainNum()) # is this correct???
+                icodes.append(resid.GetInsertionCode())
+            else:
+                warnings.warn(
+                    f"No residue is defined for atom (ID: {atom.GetIdx()})." 
+                    " Please set with 'MDA SETTING METHOD' if required." # TO DO
+                )
+                resnums.append(None) # TO DO: is this best??
+                resnames.append(None)
+                chainids.append(None)
+                icodes.append(None)
+            
+                 
+            # don't need to check null case, as know assigned to OBMol we're currently parsing
+            # but, NEED TO HANDLE ADDING MULTIPLE SEGIDS/OBMOLS WHEN CONVERTING TO UNIVERSE AND ADDING TOGETHER... (should be ok, check w tests)
+            # segids.append(atom.GetParent())
+            segids.append(0) # need better system!
+
+            # TO DO: may need to create seperate for if SMILES input
+            chirality = None
+            if atom.IsPositiveStereo():
+                chirality = "+"
+            if atom.IsNegativeStereo():
+                chirality = "-"
+            chiralities.append(chirality)
+
+            aromatics.append(atom.IsAromatic()) #boolean
+
+            # altlocs.append()
+            # occupancies.append()
+            # tempfactors.append()
+
+
+        # make Topology attributes
+        attrs = []
+        n_atoms = len(ids)
+
+        if resnums and (len(resnums) != n_atoms): #resnums.__contains__(None):
+            raise ValueError(
+                "ResidueInfo is only partially available in the molecule."
+            )
+
+        # * Attributes always present *
+
+        # Atom attributes
+        for vals, Attr, dtype in (
+            (ids, Atomids, np.int32),
+            (elements, Elements, object),
+            (masses, Masses, np.float32),
+            (aromatics, Aromaticities, bool),
+            (chiralities, RSChirality, 'U1'),
+        ):
+            attrs.append(Attr(np.array(vals, dtype=dtype)))
+
+        # Bonds
+        bonds = []
+        bond_types = []
+        bond_orders = []
+        for bond_idx in range(0, mol.NumBonds()):
+            bond = mol.GetBond(bond_idx)
+            print(bond)
+            bonds.append((bond.GetBeginAtomIdx(), bond.GetEndAtomIdx()))
+            bond_orders.append(bond.GetBondOrder()) # is int, not double. Does this matter?
```

Comment:
@hmacdope @exs-cbouy I see the RDKitParser uses double. Thoughts? I would convert to double to be safe.

Reply:
Yeah better to be consistent.